src/current/_includes/releases/v23.2/v23.2.0.md

```diff
@@ -385,11 +385,11 @@ In CockroachDB Self-Hosted, all available features are free to use unless their
 Before [upgrading to CockroachDB v23.2]({% link v23.2/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as key cluster setting changes, and adjust your deployment as necessary.
 
 - The pre-v23.1 output produced by `SHOW RANGES`, `crdb_internal.ranges`, and `crdb_internal.ranges_no_leases` was deprecated in v23.1 and is now replaced by default with output that's compatible with coalesced ranges (anges that pack multiple tables/indexes/partitions into individual ranges). See the [v23.1 release notes]({% link releases/v23.1.md %}) for `SHOW RANGES` for more details. [#102961][#102961]
```

Comment:
All of the reference-style links are not resolving as their definitions are not present in this file.

I lean toward leaving the links and copying those definitions into this file, even though we do say to go to the release notes for more information about any bullet. But typically we don't have more detail in the release notes. So I'm OK with keeping these links.

(Side note: perhaps we should go back and add links to docs for key pre-release notes, if we can find an efficient way to do so, right? Something worth considering for 24.1?)

Reply:
Done. I also added the required links to the upgrade topic. Not sure about adding links to pre-existing notes or what the process should be.